tachyon/crypto/commitments/fri/two_adic_multiplicative_coset.h

```diff
@@ -118,6 +118,12 @@ class TwoAdicMultiplicativeCoset {
     std::vector<F> last_row(sz);
     std::vector<F> transition(sz);
     std::vector<F> inv_zeroifier(sz);
+    OMP_PARALLEL_FOR(size_t i = 0; i < sz; ++i) {
+      first_row[i] = F::Zero();
+      last_row[i] = F::Zero();
+      transition[i] = F::Zero();
+      inv_zeroifier[i] = F::Zero();
+    }
```

Comment:
We can remove this, right? It seems like parallel loop below fills the values. But just add a comment that why this is safe.